src/pages/MyPage/EditProfession.js

```diff
@@ -1,31 +1,52 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
+import axios from 'axios';
 import { CATEGORY } from './PROFESSION_CATEGORY';
+import { SUBCATEGORY_ID, CAREER_ID, CAREER_ID_TO_STR } from './MYPAGE_IDS';
 
-const EditProfession = ({ setIsProfessionEdit }) => {
-  const [selectedOptions, setSelectedOptions] = useState({
-    category: '',
-    subcategory: '',
-    years: '',
-    salary: '',
-  });
+const EditProfession = ({ info, setIsProfessionEdit, token, setIsUpdated }) => {
+  const initialSelectedOptions = {
+    category: `${info.job_category ? info.job_category : ''}`,
```

Comment:
```suggestion
    category: info.job_category || ""
```

삼항연산자 안쓰고 or 연산자만으로도 표현 가능한 케이스 같네요!